### PR TITLE
Added BingApi - SnapToRoute

### PR DESCRIPTION
**Pull request by carloscaldas**
Closes https://github.com/LBNL-UCB-STI/beam/issues/3068

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3072)

### CODE REVIEW

File: src/main/scala/beam/utils/mapsapi/bingapi/BingExampleOfUsage.scala [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package beam.utils.mapsapi.bingapi
```
Comment (JustinPihony) [inline src/main/scala/beam/utils/mapsapi/bingapi/BingExampleOfUsage.scala:1]: This seems a bit hidden - should it be referenced in the docs somewhere?

**Review by JustinPihony — APPROVED** [review]
Minor items

File: src/main/scala/beam/utils/mapsapi/bingapi/BingExampleOfUsage.scala [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import scala.concurrent.duration.FiniteDuration
+import scala.util.Try
+
+import beam.agentsim.infrastructure.geozone.WgsCoordinate
+import beam.utils.FileUtils
+
+object BingExampleOfUsage extends App {
+  if (args.length != 2) {
+    println("Expected arguments: [API-KEY] [lat1,long1;lat2,long2;lat3,long3]")
+    println("Example: KqkuBonCHeDLytZwdGfKcUH9N287H-lOdqu 37.705687,-122.461096 37.724113,-122.447652")
```
[the remark this answers is not in the document]
Comment (carloscaldas) [reply src/main/scala/beam/utils/mapsapi/bingapi/BingExampleOfUsage.scala]: fixed

File: src/main/scala/beam/utils/mapsapi/bingapi/BingExampleOfUsage.scala (hunk 1 above)
Comment (carloscaldas) [reply src/main/scala/beam/utils/mapsapi/bingapi/BingExampleOfUsage.scala:1]: we already have two other examples: here-api, bing-api and google-api. we probable should define and create another task for updating all of them

**Comment by carloscaldas** [comment]
Test now!

**Comment by carloscaldas** [comment]
Test!

**Comment by carloscaldas** [comment]
Test now!

**Comment by carloscaldas** [comment]
Test!

**Comment by carloscaldas** [comment]
Test!

**Comment by carloscaldas** [comment]
Test now!

**Pull request merged**